### ISSUE TITLE
Test GHCR

### ISSUE THREAD

**Issue by UncleGedd** [defenseunicorns/uds-cli#43]
As a UDS-CLI dev
I want to test Bundles with Zarf packages that are in GHCR
So I can be confident UDS-CLI is compatible with GHCR

**Notes**
- Recommend pulling the Zarf init pkg from GHCR as part of `src/test/packags/01-uds-bundle` and using that in one of the existing tests

**Comment by anthonywendt** [comment]
Attempted to use `src/test/packags/01-uds-bundle` but ran into some issues.

I think the use of the `--insecure` flag will need to be reworked a bit?

I found that I couldn't create a bundle with ghcr because we were using the `--insecure` flag hard coded during create in the tests. That was causing issues with authentication to ghcr.

I also couldn't use a combination of an insecure zarf package from a local registry along with a secure ghcr package in the same bundle. Should these `--insecure` things be handled differently? New issue?

I updated test 3 to use the ghcr init package and a local pod info package. Seemed the best fit for this for now.
